app/client/cli/debug.go

```diff
@@ -44,18 +47,27 @@ var (
 		PromptSendBlockRequest,
 	}
 
-	// validators holds the list of the validators at genesis time so that we can use it to create a debug address book provider.
-	// Its purpose is to allow the CLI to "discover" the nodes in the network. Since currently we don't have churn and we run nodes only in LocalNet, we can rely on the genesis state.
-	// HACK(#416): This is a temporary solution that guarantees backward compatibility while we implement peer discovery
-	validators []*coreTypes.Actor
-
 	configPath  string = runtime.GetEnv("CONFIG_PATH", "build/config/config1.json")
 	genesisPath string = runtime.GetEnv("GENESIS_PATH", "build/config/genesis.json")
+	rpcHost     string
 )
 
+// NOTE: this is required by the linter, otherwise a simple string constant would have been enough
+type cliContextKey string
+
+const busCLICtxKey = "bus"
```

Comment:
Re: https://github.com/pokt-network/pocket/pull/491/files#r1107988965

I simplified using context only to pass around the `bus` and then we use it as usual

Reply:
🚌